README.md

```diff
@@ -10,32 +10,49 @@ GnuCash invoice printer for human beings.
 
 ## Usage
 
-GnuCash invoice printer currently supports SQLite and the usage is dead simple.
+GnuCash invoice printer currently supports SQLite and MySQL (gnucash-3.x) and
+the usage is dead simple.
+
+    $ gnucash-invoice --help
+
+
 List all invoices that can be printed:
 
-    $ gnucash-invoice --dbpath /path/to/db.sqlite3
+    $ gnucash-invoice --db-path /path/to/db.sqlite3
+
+or:
+
+    $ gnucash-invoice -a mysql -h db.somewhere.net -d dbname -u user -p sekreet
```

Comment:
Probably it's better to change to simply accept `--db` argument that will be a URL like:

```
sqlite:///path/to/db.sqlite3
mysql://user:pass@db.somewherenet/dbname
```

Reply:
Not saying I insist. Just a thought,